aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -106,6 +106,7 @@ spec aptos_framework::transaction_validation {
         txn_expiration_time: u64,
         chain_id: u8,
     ) {
+        pragma timeout = 100;
```

Comment:
?

Reply:
Removed. It was added because the spec didn't pass.